src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <v-card flat id="relationship-style">
+    <div class="form__row three-column mt-3">
+      <v-card flat rounded="sm" class="relationship_title gray-card px-4">
```

Comment:
Please rename this class. It's not a title. Also we don't use snake-case for classnames. You could name is "relationship-vcard".

Reply:
You'll need to add "gray-card" class to the styles (assets) in this repo.

Reply:
Fixed. I cleaned up my styles and code. I removed the unnecessary div. I also removed the gray-card. Now, the styles are specific to this component via relationships-panel and the relationships-vcard class.

Reply:
It's way cleaner. Thanks!